editoast/editoast_derive/src/model/parsing.rs

```diff
@@ -292,3 +296,76 @@ impl FieldTransformation {
         }
     }
 }
+
+impl Errors {
+    fn from_args(args: ErrorArgs) -> darling::Result<Self> {
+        match args {
+            ErrorArgs::CatchAll(path) => Ok(Self {
+                create: path.clone(),
+                retrieve: path.clone(),
+                update: path.clone(),
+                delete: path,
+            }),
+            ErrorArgs::Fine(FineErrorArgs {
```

Comment:
How can we be sure that we are exhaustive in catching all problematic situations? What about `create` and `delete` for example?

Reply:
I completely misunderstood this part. But I discussed it with @leovalais and it will probably change in a future version. I'm resolving since the original commit doesn't make sense.

Reply:
I got a few more comments on this improvements... But I made separate threads.